include/seastar/rpc/rpc.hh

```diff
@@ -616,6 +617,7 @@ public:
     server(protocol_base* proto, server_socket, resource_limits memory_limit = resource_limits(), server_options opts = server_options{});
     server(protocol_base* proto, server_options opts, server_socket, resource_limits memory_limit = resource_limits());
     void accept();
+    future<> shutdown();
     future<> stop();
```

Comment:
Please add documentation. Yes, rpc is undocumented and the policy is to only update documented interfaces, but stop() and shutdown() are too similar and must be distinguished by an explanation.